k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/model/K8sManifests.kt

```diff
@@ -10,8 +12,13 @@ data class K8sData(
     val account: String? = null,
     val username: String? = null,
     val password: String? = null,
-    val identity: String? = null
-)
+    val identity: String? = null,
+    val known_hosts: String? = null
+) {
+    fun toMap(): Map<String, String?> {
+        return jacksonObjectMapper().convertValue(this, object: TypeReference<Map<String, String?>>() {})
+    }
+}
```

Comment:
we can probably get rid of `K8sData` all together. It doesn't provide much value anymore.